.npmignore

```diff
@@ -0,0 +1,7 @@
+.eslintrc
+.idea
+.prettierrc
+.env
+.env.sample
+scripts
+src/index.html
```

Comment:
I think it's safer to use the `files` property in `package.json` for this ([docs](https://docs.npmjs.com/cli/v9/configuring-npm/package-json#files)). It achieves the same, it only works as include instead of exclude. Using both `.gitignore` and `.npmignore` together can create problems if I remember correctly

Reply:
The `files` entry seems to be checked when the package is installed, `.npmignore` seems to be checked when the package is built. I think it's more sensible to keep files out of the package by default.

Reply:
Seems to have the same effect in my experience. I just think blacklisting is prone to error and I heard some bad stuff about `.npmignore` over the years. I also don't see it used very often in the wild, but I am not sure. Anyway no big deal